addons/pos_restaurant/static/tests/tours/ticket_screen_tour.js

```diff
@@ -17,40 +17,14 @@ registry.category("web_tour.tours").add("PosResTicketScreenTour", {
             TicketScreen.noNewTicketButton(),
             TicketScreen.clickDiscard(),
 
-            // Deleting the last order in the table brings back to floorscreen
-            FloorScreen.clickTable("4"),
-            ProductScreen.isShown(),
-            Chrome.clickMenuOption("Orders"),
-            TicketScreen.nthRowContains(2, "-0001"),
-            TicketScreen.deleteOrder("-0001"),
-
-            // Create 2 items in a table. From floorscreen, delete 1 item. Then select the other item.
-            // Correct order and screen should be displayed and the BackToFloorButton is shown.
-            FloorScreen.clickTable("2"),
-            ProductScreen.addOrderline("Minute Maid", "1", "2"),
-            ProductScreen.totalAmountIs("2.0"),
-            Chrome.clickMenuOption("Orders"),
-            TicketScreen.clickNewTicket(),
-            ProductScreen.addOrderline("Coca-Cola", "2", "2"),
-            ProductScreen.totalAmountIs("4.0"),
-            Chrome.clickPlanButton(),
-            FloorScreen.orderCountSyncedInTableIs("2", "3"),
-            Chrome.clickMenuOption("Orders"),
-            TicketScreen.deleteOrder("-0003"),
-            Dialog.confirm(),
-            TicketScreen.doubleClickOrder("-0002"),
-            ProductScreen.isShown(),
-            ProductScreen.totalAmountIs("2.0"),
-            Chrome.clickPlanButton(),
-
```

Comment:
I don't why you have to remove these test steps.

But if they are not really necessary, instead of remove, I think it's better to replace them by testing the newly introduced control button.

Reply:
this tests that we handle correctly the case when there are multiple orders on the same table, but that is a functionality that we no longer plan to support. That's why i removed the steps.

There is still a tour that tests the deletion, though ( `pos_restaurant_sync` )